lib/ok.ex

```diff
@@ -72,65 +49,51 @@ defmodule OK do
   end
 
   @doc """
-  Result pipe operator.
-  (Result monad bind operator)
-
-  The result pipe takes the value out of an `{:ok, value}` tuple and passes it as the first argument to the function call on the right.
-
-  ## Examples
-
-      iex> {:ok, 5} ~>> double()
-      {:ok, 10}
+  The OK result pipe operator `~>>`, or result monad bind operator, is similar to Elixir's native `|>` except it is used within happy path. It takes the value out of an `{:ok, value}` tuple and passes it as the first argument to the function call on the right.
 
-      iex> {:error, :previous_bad} ~>> double()
-      {:error, :previous_bad}
-
-      # x is {:ok, 7} defined in `OKTest`.
-      iex> x() ~>> double()
-      {:ok, 14}
-
-  The result pipe is most useful when executing a series of operations that may fail.
-
-      iex> {:ok, 6} ~>> safe_div(3) ~>> double
-      {:ok, 4.0}
-
-      iex> {:ok, 6} ~>> safe_div(0) ~>> double
-      {:error, :zero_division}
 
   It can be used in several ways.
   Pipe to a local call.
   This example is the same as calling `double(5)`
 
-      iex> {:ok, 5} ~>> double
-      {:ok, 10}
+    iex> {:ok, 5} ~>> double()
```

Comment:
Ah, interesting. It didn't even occur to me Doh! I will definitely do that. :+1: 

Did you have a chance to checkout why the hexdocs lost the existing module docs?